rnndb/fifo/gf100_pfifo.xml

```diff
@@ -217,7 +217,14 @@
 		<bitfield pos="8" name="ACTIVE"/>
 	</reg32>
 	<array offset="0x800" name="VM_FAULT" length="32" stride="0x10">
-		<reg32 offset="0" name="CHAN" type="g80_channel"/>
+		<reg32 offset="0" name="CHAN" type="g80_channel">
```

Comment:
Don't list bitfields in this register - setting type="g80_channel" already does that.